extension/src/Utils/classPathUtility.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as Configs from '../configs';
+import { Logger } from '../logger';
+
+import * as archiver from 'archiver';
+import * as fileUrl from 'file-url';
+import * as fs from 'fs';
+import * as mkdirp from 'mkdirp';
+import * as os from 'os';
+import * as path from 'path';
+
+export class ClassPathUtility {
+    // TODO: refactor logger, no need to pass in logger instance.
+    public static processLongClassPath(

```

Comment:
The name `processLongClassPath` is not explicit regarding its purpose. Code should be self-describing. By reading the implementation, `generateClassPathFile` is the appropriate name.